File: PRAKTIKUM 5/PRAK501_2310817220028_SheilaSabina.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d);

int main() {
    int a, b, c, d; 
    printf("Input\n");
    scanf("%d %d %d %d", &a, &b, &c, &d);

    int hasil = MaxBilangan(a, b, c, d);
    printf("\nOutput\n");
    printf("%d", hasil); 
    return 0;
}

int MaxBilangan(int a, int b, int c, int d) {
    int maksimum = a;

    if (b > maksimum) {
        maksimum = b;
    }

    if (c > maksimum) {
        maksimum = c;
    }

    if (d > maksimum) {
        maksimum = d;
    }

    return maksimum;
}

File: PRAKTIKUM 5/PRAK501_2310817220028_SheilaSabina.py
def max_bilangan(a, b, c, d):
    maksimum = a

    if b > maksimum:
        maksimum = b

    if c > maksimum:
        maksimum = c

    if d > maksimum:
        maksimum = d

    return maksimum

if __name__ == "__main__":
    print("Input")
    a, b, c, d = map(int, input().split())

    hasil = max_bilangan(a, b, c, d)
    
    print("\nOutput")
    print(hasil)